assets/xml/archives/icon_item_static.xml

```diff
@@ -47,8 +47,8 @@
         <Texture Name="gItemIconEmptyBottle2Tex" OutName="empty_bottle_2_icon" Format="rgba32" Width="32" Height="32" Offset="0x27000" />
 
         <!-- Trade Quest icons -->
-        <Texture Name="gItemIconLandDeedTex" OutName="land_deed_icon" Format="rgba32" Width="32" Height="32" Offset="0x29000" />
-        <Texture Name="gItemIconMoonsTearTex" OutName="moons_tear_icon" Format="rgba32" Width="32" Height="32" Offset="0x28000" />
+        <Texture Name="gItemIconMoonsTearTex" OutName="moons_tear_icon" Format="rgba32" Width="32" Height="32" Offset="0x29000" />
+        <Texture Name="gItemIconLandDeedTex" OutName="land_deed_icon" Format="rgba32" Width="32" Height="32" Offset="0x28000" />
```

Comment:
```suggestion
        <Texture Name="gItemIconMoonsTearTex" OutName="moons_tear_icon" Format="rgba32" Width="32" Height="32" Offset="0x28000" />
        <Texture Name="gItemIconLandDeedTex" OutName="land_deed_icon" Format="rgba32" Width="32" Height="32" Offset="0x29000" />
```
The offsets were attached to the right item, it's just that they were in the wrong order in the xml, so you have to switch the whole line, not just the offset in isolation

Reply:
Oh! Imma dummy 😂 I'll do that, thanks!